frame/assets/src/lib.rs

```diff
@@ -1601,6 +1600,21 @@ pub mod pallet {
 			Self::do_refund_other(id, &who, &origin)
 		}
 	}
+
+	/// Implements `AccountTouch` trait.
```

Comment:
Please use proper links, in light of the 4th principle named [here](https://forum.parity.io/t/new-polkadot-substrate-documentation-plan/1810) (still working on writing a guideline for how to document FRAME code, but one is indeed using links where possible).